scripts/mirna_extension.py

```diff
@@ -23,15 +23,12 @@
 import os
```

Comment:
Do you still need this? If so, see if you can use `pathlib.Path` instead. No point in importing a rather large module if we don't really need it.